docs/build-and-publish-pipeline.md

```diff
@@ -0,0 +1,68 @@
+## Application Services Build and Publish Pipeline
```

Comment:
Could we make a table of contents or an index of docs under https://github.com/mozilla/application-services/tree/master/docs/howtos ? I can never properly navigate the how-tos by looking at this view http://v14d.com/i/0ccf88a289f1336dcb04769576ad23ad60c1cedd4460ccce3d6d65c8a96693aa.jpg Am I doing it wrong?

Reply:
👍, yes, excellent suggestion.